cmd/vcfAfs/vcfAfs_test.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fileio"
+	"os"
+	"testing"
+)
+
+var VcfAfsTests = []struct {
+	inputFile               string
+	expectedFile            string
+	unPolarized             bool
+	plotSelectionLikelihood string
+	leftBound               float64
+	rightBound              float64
+	numberOfPoints          int
+	integralError           float64
+}{
+	{"testdata/simulate.N100.S100.Seed19.Alpha0.01.vcf", "testdata/test.afs.txt", true, "", -10, 10, 100, 1e-5},
```

Comment:
Would it be good to have at least one other input instance to test? Maybe one is enough, idk?